.github/workflows/welcome-newcomers.yaml

```
@@ -0,0 +1,40 @@
+name: first-interaction
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    branches: [main]
+    types: [opened]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Welcome to the Pantsbuild Community. This looks like your first issue here.
```

Maybe worth also acknowledging that they took time to make an issue?

```suggestion
            Welcome to the Pantsbuild Community. This looks like your first issue here. Thanks for taking the time to write it.
```

or something.